src/Bridge/NumPy/ConvertNumPyArrayToitkImage/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.16.3)
+
+project( ConvertNumPyArrayToitkImage )
+
+find_package( ITK REQUIRED )
+include( ${ITK_USE_FILE} )
+
+if(ITK_WRAP_PYTHON)
+  enable_testing()
+  add_test(NAME  ConvertNumPyArrayToitkImageTestPython
+    COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/Code.py
+    Slice.png
+    SliceRevised.png
+    )
```

Comment:
These files need to be prefixed with `${CMAKE_CURRENT_BINARY_DIR}/` so that CTest gets the absolute path to each image file.